app/Models/Language.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Model;
+use Illuminate\Database\Eloquent\Concerns\HasUuids;
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+
+class Language extends Model
+{
+    use HasFactory;
+    use HasUuids;
+
+    protected $guarded = ['id'];
+    public $timestamps = false;
```

Comment:
You found the solution!!!! :clap: :clap:

Reply:
I'm not sure if it's the best solution, but it works. Apart from that, we owe the "magic" to  @FranEnLaNube

Reply:
Yeah, this is a good solution, but we need to keep an eye on the current situation, as there is currently no registration system in place. Future problems... haha, now this is fine for us :muscle: